src/macros.rs

```diff
@@ -38,9 +40,17 @@ macro_rules! redis_command {
         {
             return $crate::raw::Status::Err as c_int;
         }
+        let context = $crate::Context::new($ctx);
```

Comment:
Nit: Can this be moved into the block at line 46?

Reply:
Fixed, now is in the block